src/analytics/index.ts

```diff
@@ -27,6 +29,13 @@ export function initializeAnalytics(apiKey: string, originApplication: OriginApp
   if (isInitialized) {
     throw new Error('initializeAnalytics called multiple times - is it inside of a React component?')
   }
+
+  if (config?.debug && config.isProductionEnv) {
+    throw new Error(
+      `It looks like you're trying to initialize analytics in debug mode for production. Please disable debug mode or the production environment`
```

Comment:
**on** the production environment?

Reply:
Correct as is - in this case it's saying you can disable the debug mode OR the production environment to stop the error. Depending on the context the fix changes here (locally debugging accidentally with production on vs accidentally committing debug = true)